arangod/IResearch/IResearchRocksDBRecoveryHelper.cpp

```diff
@@ -216,7 +216,8 @@ void IResearchRocksDBRecoveryHelper::LogData(rocksdb::Slice const& blob,
   auto const type = RocksDBLogValue::type(blob);
 
   switch (type) {
-    case RocksDBLogType::IndexCreate: {
+    case RocksDBLogType::IndexCreate:
+    case RocksDBLogType::IndexDrop: {
```

Comment:
Why Drop?

Reply:
Because it invalidates pointer in the range (on practice everything will be ok because we're holding reference in the post recovery callback, but we will just execute unnecessary inserts/removes)